examples/monaco.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <!-- Firebase -->
+    <script src="https://www.gstatic.com/firebasejs/3.3.0/firebase.js"></script>
+
+    <!-- Firepad -->
+    <link rel="stylesheet" href="https://cdn.firebase.com/libs/firepad/1.4.0/firepad.css" />
+    <script src="https://cdn.firebase.com/libs/firepad/1.4.0/firepad.min.js"></script>
```

Comment:
Obviously this doesn't work yet.  I can change it to 1.5.0 once it's released...  but given that as-written, this depends on local files (node_modules/monaco-editor) you could also just change this to ../dist/firepad.min.js so that the example works as-is.